taskcluster/tc-asserts.sh

```diff
@@ -524,6 +524,15 @@ run_multi_inference_tests()
   assert_correct_multi_ldc93s1 "${multi_phrase_pbmodel_withlm}" "$status"
 }
 
+run_hotword_tests()
```

Comment:
maybe make a `run_android_hotword_tests` with android specific paths like `DATA_TMP_DIR`, see `run_tflite_basic_inference_tests` for more insights.

Reply:
added